src/lumigo_tracer/sync_http/sync_hook.py

```diff
@@ -164,6 +165,7 @@ def lambda_wrapper(*args, **kwargs):
         try:
             if Configuration.enhanced_print:
                 _enhance_output(args, local_print, local_logging_format)
+            AutoTagEvent.auto_tag_event(*args)
```

Comment:
I think that `args[0]` is `event` and `args[1]` is `context`.
But the second argument of `auto_tag_event` isn't `context`. 🤔

Reply:
Didn't understand the issue ?